packages/components/src/Components/Inventory/AppInfo.js

```diff
@@ -17,20 +22,23 @@ const AppInfo = React.forwardRef((props, ref) => {
                 appName="chrome"
                 module="./AppInfo"
                 scope="chrome"
-                ErrorComponent={<AsyncInventory ref={ref} component="AppInfo" {...props} />}
-                ref={ref}
+                ErrorComponent={<AsyncInventory component="AppInfo" {...props} />}
+                ref={props.innerRef}
                 {...props}
             />
         </Suspense>
     );
-});
+};
 
-AppInfo.propTypes = {
+BaseAppInfo.propTypes = {
+    /** React Suspense fallback component. <a href="https://reactjs.org/docs/code-splitting.html#reactlazy" target="_blank">Learn more</a>. */
     fallback: PropTypes.node
 };
 
-AppInfo.defaultProps = {
+BaseAppInfo.defaultProps = {
     fallback: <Bullseye><Spinner size="xl" /></Bullseye>
 };
 
+const AppInfo = React.forwardRef((props, ref) => <BaseAppInfo innerRef={ref} {...props} />);
```

Comment:
I know it's not important right now but just a note that this will mess up the new docs generator because the props will be missing.

Reply:
I should actually change the prop definition as well, because consumers will use these when using this.